static/i18n/en.lua

```diff
@@ -19,5 +18,29 @@ return {
     administrator = "administrator",
     user_created = "User created",
     error_header = "ERROR",
+    navigation_title = "Navigation links",
+    upload_button_header = "Select package to upload",
+    choose_file = "Choose Package File",
+    upload_package = "Upload Package",
+    package_name = "Package name",
+    package_version = "Version",
+    package_description = "Description",
+    package_game = "Game package is intended for",
+    package_dependencies = "Dependencies. package names separated by spaces",
```

Comment:
Hmm so you can't have a package with a space in the name? That seems rather rough.

I don't know how other sites do it, but we could just serialise a Lua table into the text db field.

Reply:
That's easy enough to change, it's only a text description for now.

Reply:
what about comma separated values?

Reply:
Should be fine, as long as you follow the CSV specs so values can indeed contain commas, quotes, etc. as well

Reply:
all I'm doing on this side is storing the value. It's the client side that'll have to parse the string and worry about all that, I think. We can work to ensure the client side respects escaped commas and the like.